docs/http4s.md

```diff
@@ -0,0 +1,93 @@
+# Module http4s
+
+[![Maven Central](https://img.shields.io/maven-central/v/com.avast/scala-server-toolkit-http4s-blaze-server_2.12)](https://repo1.maven.org/maven2/com/avast/scala-server-toolkit-http4s-blaze-server_2.12/)
+
+`libraryDependencies += "com.avast" %% "scala-server-toolkit-http4s-blaze-server" % "<VERSION>"`
+
+There are `http4s-*` modules that provide easy initialization of a server and a client. Http4s is an interface with multiple possible
+implementations - for now we provide only implementations based on [Blaze](https://github.com/http4s/blaze).
+
+Both server and client are configured via configuration `case class` which contains default values taken from the underlying implementations.
+Initialization of a server and a client is very simple if defaults are fine for you.
```

Comment:
This sentence IMHO doesn't bring any value so I would remove it.

Reply:
Alright.